exist-core/src/main/java/org/exist/util/Configuration.java

```diff
@@ -98,36 +97,30 @@
 import static javax.xml.XMLConstants.FEATURE_SECURE_PROCESSING;
 
 
-public class Configuration implements ErrorHandler
-{
-    private final static Logger       LOG            = LogManager.getLogger(Configuration.class); //Logger
-    protected Optional<Path>          configFilePath = Optional.empty();
-    protected Optional<Path>          existHome      = Optional.empty();
-
-    protected DocumentBuilder         builder        = null;
-    protected HashMap<String, Object> config         = new HashMap<>(); //Configuration
-
+public class Configuration implements ErrorHandler {
+    public static final String BINARY_CACHE_CLASS_PROPERTY = "binary.cache.class";
+    private static final Logger LOG = LogManager.getLogger(Configuration.class); //Logger
     private static final String XQUERY_CONFIGURATION_ELEMENT_NAME = "xquery";
     private static final String XQUERY_BUILTIN_MODULES_CONFIGURATION_MODULES_ELEMENT_NAME = "builtin-modules";
     private static final String XQUERY_BUILTIN_MODULES_CONFIGURATION_MODULE_ELEMENT_NAME = "module";
-    
-    public final static String BINARY_CACHE_CLASS_PROPERTY = "binary.cache.class";
-    
+    protected Optional<Path> configFilePath = Optional.empty();
+    protected Optional<Path> existHome = Optional.empty();
+    protected DocumentBuilder builder = null;
```

Comment:
1. Should be marked `@Nullable`
2. Why `protected` and not `private`?
3. Is this actually used anywhere?

Reply:
No idea.

Reply:
@line-o If the visibility can be reduced without side effects I would do so..